embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj

```diff
@@ -5,8 +5,16 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
+  <PropertyGroup>
+    <TizenApiVersion Condition="'$(TizenApiVersion)' == ''">4.0</TizenApiVersion>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(TizenApiVersion) &lt; 6.5">
```

Comment:
I think that's enough now. but this is just a reference for later. 
https://docs.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2019#msbuild-version-comparison-functions

Reply:
Oh, the functions look cleaner and more reliable.